state_of_css.yml

```
@@ -47,11 +47,29 @@ translations:
     - key: sections.animations_transforms.description
       t: Animace a transformace prvků.
 
+    - key: sections.accessibility.title
+      t: Přístupnost
+    - key: sections.accessibility.description
+      t: Prvky a techniky v přístupnosti.
+
     - key: sections.media_queries.title
       t: Media Queries
     - key: sections.media_queries.description
       t: Přizpůsobení stránky podle zařízení nebo preferencí uživatele.
 
+    - key: sections.selectors.title
+      t: Selektory
+    - key: sections.selectors.description
+      t: CSS selektory.
+    - key: sections.colors.title
+      t: Barvy
+    - key: sections.colors.description
+      t: Vlastnosti související s barvami.
+    - key: sections.javascript_apis.title
+      t: JavaScript APIs
```

```suggestion
      t: JavaScript API
```

or

```suggestion
      t: Javascriptová API
```

Also see below.